Makefile

```diff
@@ -10,3 +10,6 @@ cpplint:
 
 cppformat:
 	clang-format --verbose -i $(CPPSOURCES)
+
+cppbuild:
+	cd ./superbench/benchmarks/; bash build.sh
```

Comment:
change `;` to `&&`? why not `./superbench/benchmarks/build.sh` directly?

Reply:
Both makefile and build.sh use relative path. If use `./superbench/benchmarks/build.sh`, the script will loop `./superbenchmark/` dir not `./superbenchmark/superbench/benchmarks`